### PR TITLE
People detection nodelet

### PR DESCRIPTION
**Pull request by iory**
Add people detection nodelet.
Refer to the following link.
http://pointclouds.org/documentation/tutorials/ground_based_rgbd_people_detection.php
people_tracking

### CODE REVIEW

File: jsk_pcl_ros/include/jsk_pcl_ros/people_tracking.h [hunk 1]
```diff
@@ -0,0 +1,117 @@
+// -*- mode: c++ -*-
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2014, JSK Lab
```
Comment (wkentaro) [inline jsk_pcl_ros/include/jsk_pcl_ros/people_tracking.h]: year

Comment (iory) [inline jsk_pcl_ros/include/jsk_pcl_ros/people_tracking.h]: OK! fixed

**Comment by wkentaro** [comment]
@furushchev Do you know what's the difference between this nodelet and the people detector nodelet used on PR2.

**Comment by iory** [comment]
Please merge this.

**Comment by k-okada** [comment]
@furushchev as far as I know pr2 uses "face detection" where as this not used "full body" detection. Is it correct?

**Comment by furushchev** [comment]
@k-okada @wkentaro Sorry for late response. I overlooked.
No, in PR2 we can know only face and 2D pose with id.
In pr2 we use https://github.com/wg-perception/people, where people is detected by face RGB-D and leg, and in JSKPR2 we added hark integration.

**Comment by k-okada** [comment]
ok, so let's compare these two package > @iory

--
◉ Kei Okada

**Pull request merged**